common.yml

```diff
@@ -2485,17 +2517,17 @@ github_pr_aliases:
   - variant: ".*"
     task: "^push$"
   # Run tests with the race detector enabled.
-  - variant: "ubuntu-race"
+  - variant: "rhel88-race"
     task: "^.*$"
   # Run all integration tests on one variant. We pick RHEL 8.0 because it's
   # a relatively recent platform that supports a wide range of servers.
-  - variant: "rhel88"
+  - variant: "rhel80"
```

Comment:
I think the comment isn't correct since we moved all the tests to rhel88.

Reply:
Argh. I will update this to add a comment to tell people to stop editing this by hand.